module/VuFindSearch/src/VuFindSearch/Backend/Solr/Response/Json/Spellcheck.php

```diff
@@ -141,9 +142,9 @@ public function mergeWith(Spellcheck $spellcheck)
     /**
      * Return aggregated iterator.
      *
-     * @return ArrayIterator
+     * @return Iterator
      */
-    public function getIterator(): \Traversable
+    public function getIterator(): Iterator
```

Comment:
Is there a strong reason to use Iterator instead of Traversable here? This is a BC-breaking change; I don't mind making and documenting it if there's a reason, but if we can avoid it, it would spare us a changelog note. :-)

Reply:
Fixed